content/en/continuous_integration/troubleshooting.md

```diff
@@ -159,13 +159,13 @@ If you have admin access, you can update it from the [Repository Settings Page][
 
 - Your repository needs to have a commit history of at least two commits in the past month.
 - You need to have collected test code coverage in past commits, which happens on test runs where Intelligent Test Runner was enabled.
-- You git clone must contain commit and tree history. Shallow git clones (`git clone --depth=0`) are not supported.
+- Your git clone must contain commit and tree history. Intelligent Test Runner tries to unshallow git clones that do not contain history (`git clone --depth=0`), but that might not work on older versions of git. If your CI job is using shallow git clones, you can change it to use partial git clones by using the following command: `git clone --filter=blob:none`.
```

Comment:
Mmm I'm not entirely understanding how they can change it. Usually the cloning part is done via the checkout GH action, right?

Reply:
How to change the cloning of the repo is going to be CI-specific, so there are no specific instructions here. Just showing what ultimately needs to be achieved, but if your cloning command is wrapped with a circleci orb or whatever then you need to go to the documentation for that to see how to proceed.

Reply:
Makes sense 👍